### PR TITLE
Helm release KubeCon blog

### PR DESCRIPTION
**Pull request by sabre1041**
Added blog for Helm release for KubeCon NA 2024

### CODE REVIEW

File: content/en/blog/2024-11-08-experience-helm-release-kubecon-na-24.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Experience a Helm Release: Live at KubeCon + CloudNativeCon North America 2024!"
+slug: "experience-helm-release-kubecon-na-24"
+authorname: "Andrew Block"
+author: "@sabre1041"
+authorlink: "https://mattfarina.com"
```
Comment (mattfarina) [inline content/en/blog/2024-11-08-experience-helm-release-kubecon-na-24.md:6]: I think this is the wrong author link 😉

**Review by mattfarina — CHANGES_REQUESTED** [review]
One nit

File: content/en/blog/2024-11-08-experience-helm-release-kubecon-na-24.md (hunk 1 above)
Comment (sabre1041) [reply content/en/blog/2024-11-08-experience-helm-release-kubecon-na-24.md]: you get all the credit that you can get!

Updated authorlink

**Review by mattfarina — APPROVED** [review]
Since the date is the 8th, this needs to be published on Friday to show. We can merge then.

**Pull request merged**